src/runner.rs

```diff
@@ -1455,6 +1467,21 @@ impl ValueStack {
     fn len(&self) -> usize {
         self.sp
     }
+
+    /// Pops the last `depth` stack entries and returns them as slice.
+    ///
+    /// Stack entries are returned in the order in which they got pushed
+    /// onto the value stack.
+    ///
+    /// # Panics
+    ///
+    /// If the value stack does not have at least `depth` stack entries.
+    pub fn pop_as_slice(&mut self, depth: usize) -> &[RuntimeValueInternal] {
```

Comment:
I would suggest removing the `as` here. In my mind that means returning the whole thing as slice and not only parts of it.

Reply:
Good suggestion, will do this!